### PR TITLE
Bulk import from homebrew/versions (3/7)

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Import formulae from homebrew/versions. Those that fail to build will likely be killed.

### CODE REVIEW

**Comment by fxcoudert** [comment]
Some audits complaining about lack of test. More interesting is:

```
kibana:
  * Formula has other versions so create an alias:
      cd /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Aliases
      ln -s ../Formula/kibana.rb kibana@5.2
```

and 

```
kibana@4.1:
  * Use Language::Node for npm install args
kibana@4.4:
  * Use Language::Node for npm install args
```

File: Formula/jboss-as@5.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class JbossAsAT5 < Formula
+  desc "JBoss Application Server"
+  homepage "https://www.jboss.org/jbossas"
+  url "https://downloads.sf.net/project/jboss/JBoss/JBoss-5.1.0.GA/jboss-5.1.0.GA.zip"
```
Comment (vszakats) [inline Formula/jboss-as@5.rb]: It'd be nice to use the `downloads.sourceforge.net` domain to stay uniform.

**Comment by fxcoudert** [comment]
Merged in #11260

**Comment by MikeMcQuaid** [comment]
Thanks @fxcoudert!

**Pull request closed without merging**